CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 0.2.3-alpha.3
```

Comment:
v0.2.3-alpha.7, refer to https://github.com/bnb-chain/greenfield-storage-provider/releases .